.travis.yml

```diff
@@ -25,13 +25,6 @@ matrix:
     sudo: required
     dist: trusty
     services: docker
-    env: BEAKER_set="debian-6"
```

Comment:
We don't want to remove squeeze support for now as we still have some server not migrated yet...